backup-import/src/main/java/com/opsgenie/tools/backup/importers/ScheduleImporter.java

```diff
@@ -205,5 +207,14 @@ protected String getEntityIdentifierName(ScheduleConfig entity) {
     }
 
     @Override
-    protected void updateTeamIds(ScheduleConfig entity) {}
+    protected void updateTeamIds(ScheduleConfig entity) throws Exception {
+        Map<String, String> teamIdMap = new TeamIdMapper(rateLimitManager).getTeamIdMap();
+        TeamMeta ownerTeam = entity.getSchedule().getOwnerTeam();
```

Comment:
Do we forsee a case where entity.getSchedule() can be null and this leads to NPE?